### ISSUE TITLE
some strange bug. 

### ISSUE THREAD

**Issue by Tachii** [KoffeinFlummi/AGM#2189]
more info here: http://feedback.rhsmods.org/view.php?id=439

**Comment by Tachii** [comment]
basiacally it crashes the mission on the dedicated server and doesn't allow to start it if RHS air is used.

**Comment by Tachii** [comment]
Occurs only on dedicated server

**Comment by commy2** [comment]
The RPT file you posted in the RHS issue tracker is incomplete. Can you please upload the full RPT file?

**Comment by Tachii** [comment]
Sure, as soon as server admin will come online

**Comment by Tachii** [comment]
While he is offline, this is the most i could get from live server logs: https://dl.dropboxusercontent.com/u/110028032/log.cpp

**Comment by Tachii** [comment]
As you can see it doesnt appear immediately, it appears after some time.

**Comment by commy2** [comment]
This is no RPT-file. Seems like some 3rd party log file. Sry, but I can't help you with that.

**Comment by commy2** [comment]
Tested with Arma 1.40, RHS 0.36 and AGM 0.953 and I can't reproduce the same error.

RHS_C130J: ["RHS_C130J_Base","Plane_Base_F","Plane","Air","AllVehicles","All"]
CopilotTurret: ["MainTurret","NewTurret"]
MainTurret: ["NewTurret"]

**Comment by Tachii** [comment]
Are you testing it on dedicated server?

**Comment by commy2** [comment]
No, because that wouldnt change the classes.

Are you playing with the dev branch of Arma by any chance?

**Comment by Tachii** [comment]
No, stable branch, i can't reproduce it in singleplayer editor as well, only happens on dedicated server for some strange reason

**Comment by rlex** [comment]
i have agm + rhs and hasn't saw any problems with C-130...
